floodsub.go

```diff
@@ -8,6 +8,7 @@ import (
 	host "github.com/libp2p/go-libp2p-host"
 	peer "github.com/libp2p/go-libp2p-peer"
 	protocol "github.com/libp2p/go-libp2p-protocol"
+	"github.com/libp2p/go-libp2p-pubsub/metrics"
```

Comment:
named import please.

Reply:
I would recommend against naming it. The conventions in Go are clear, we're just being a nuisance by deviating.

Reply:
I must insist. We use them everywhere in the code and I would like to be consistent.
Also, idiomatic or not, import naming is part of the language and it makes searching for symbols much easier.

Reply:
With @vyzo  on this one as once we register metrics from another `metrics` package, i.e. dht, we are going to need to name it anyway.